app/services/analytics.rb

```diff
@@ -157,13 +157,15 @@ def browser_attributes
   IDV_DOC_AUTH_SUBMITTED_IMAGE_UPLOAD_FORM = 'IdV: doc auth image upload form submitted'.freeze
   IDV_DOC_AUTH_SUBMITTED_IMAGE_UPLOAD_VENDOR = 'IdV: doc auth image upload vendor submitted'.freeze
   IDV_DOC_AUTH_SUBMITTED_PII_VALIDATION = 'IdV: doc auth image upload vendor pii validation'.freeze
-  IDV_DOWNLOAD_PERSONAL_KEY = 'IdV: download personal key'.freeze
   IDV_FINAL = 'IdV: final resolution'.freeze
   IDV_FORGOT_PASSWORD = 'IdV: forgot password visited'.freeze
   IDV_FORGOT_PASSWORD_CONFIRMED = 'IdV: forgot password confirmed'.freeze
   IDV_INTRO_VISIT = 'IdV: intro visited'.freeze
   IDV_JURISDICTION_VISIT = 'IdV: jurisdiction visited'.freeze
   IDV_JURISDICTION_FORM = 'IdV: jurisdiction form submitted'.freeze
+  IDV_PERSONAL_KEY_DOWNLOADED = 'IdV: personal key downloaded'.freeze
```

Comment:
can you add a comment for the old one (and similar to other thread, I would just consider leaving the key the same but renaming the constant only)

```suggestion
  IDV_PERSONAL_KEY_DOWNLOADED = 'IdV: personal key downloaded'.freeze # previously "IdV: download personal key"
```

Reply:
I committed your suggestion.

The old key is not used in any terraformed dashboards, so I'm going to leave it changed to be consistent with the other personal key events.